src/components/index.ts

```diff
@@ -5,6 +5,7 @@ export { default as Tooltip } from './tooltip/Tooltip';
 export { TextInput } from './text-input/TextInput';
 export { Alert } from './alert/Alert';
 export { AlertSeverity } from './alert/types';
+export { default as Button } from './button/Button';
```

Comment:
If you keep only the named export, make sure to change this line to 
```suggestion
export { Button } from './button/Button';
```

Reply:
I keep only the default export, actually.

Reply:
Edit: finally the named exports are preferred